core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraXCameraUseCase.kt

```diff
@@ -385,13 +295,18 @@ constructor(
                 .tryApplyFrameRateConstraints()
                 .tryApplyStabilizationConstraints()
         if (isDebugMode && Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
-            val cameraProperties = getAllCamerasPropertiesJSONArray().toString()
-            val file = File(
-                Environment.getExternalStoragePublicDirectory(DIRECTORY_DOCUMENTS),
-                "JCACameraProperties"
-            )
-            writeFileExternalStorage(file, cameraProperties)
-            Log.d(TAG, "JCACameraProperties written to ${file.path}. \n" + "$cameraProperties")
+            withContext(Dispatchers.IO) {
```

Comment:
The better way is using injection with Hilt like a defaultDispatcher. If it is hard to deal with in this PR, it seems better file it into issue and makes dedicated PR.

Reply:
Agreed. You can stil use Dispatchers.IO, but it should be injected. See how `provideDefaultDispatcher` works in `CommonModule`. That way we can inject test dispatchers in tests if needed.